R/TaskClassif_melanoma.R

```diff
@@ -0,0 +1,144 @@
+#' @title Melanoma Image classification
+#' @name mlr_tasks_melanoma
+#' @description
+#' Classification of melanoma tumor images.
+#' The data is a preprocessed version of the 2020 SIIM-ISIC challenge where
+#' the images have been reshaped to size $(3, 128, 128)$.
+#'
+#' By default only the training rows are active in the task,
+#' but the test data (that has no targets) is also included.
+#'
+#' There are no labels for the test rows, so by default, these observations are inactive,
```

Comment:
New descriptions of task filtering, some extra info in the backend, and explanations of the columns (target and features) in the task